### ISSUE TITLE
Colors not shown correctly in FireFox

### ISSUE THREAD

**Issue by nippur72** [matsondawson/vic20dart#7]
If opened in FireFox, the startup screen is totally blue. 

By giving `POKE 36879,0` the screen turns blue with black border and black text.

**Comment by mattdawson** [comment]
It sounds like a issue in vic code, probably around rendering directly to bitmap data.
Maybe related to alpha.

**Comment by nippur72** [comment]
Yes, I verified and it's a kind of overflow problem with the white color `0xFFFFFFFF` of the palette. Decreasing its alpha value to `FE` makes it work correctly. 

Do you have any clue why this happens? It's a Firefox bug?

BTW, I've also found some palette entries aren't correct (e.g. light cyan), so I've replicated the palette from the VICE emulator. Sending a pull request.
